test/visual/visual.js

```diff
@@ -138,24 +149,30 @@ module.exports = {
                         `${currentDir}/${type}/${view}.png`
                     );
                     const fileSizeInBytes = stats.size;
-                    console.log(
-                        `📸 ${view}.png => ${fileSizeInBytes} bytes, ${percentDiff}% different`
-                    );
 
-                    if (numDiffPixels > 10) {
-                        diff.pack().pipe(
-                            fs.createWriteStream(
-                                `${currentDir}/${view}-diff.png`
-                            )
-                        );
-                    }
-                    // A number of comparisons come out to 10 inexplicable pixes EVERY time,
-                    // so let them have their stupid 10 pixels!
-                    expect(
-                        numDiffPixels,
-                        'number of different pixels'
-                    ).be.lessThan(11);
-                    resolve();
+                    resolve({
+                        // Return the diff file creation and file comparisin expect so that the "test"
+                        // can be run only when it "passes" or is the second try.
+                        test: () => {
+                            console.log(
+                                `📸 ${view}.png => ${fileSizeInBytes} bytes, ${percentDiff}% different`
+                            );
```

Comment:
it would be great if we could get this logged out on failures as well (and persisted as an artifact somewhere so we can review where these blips are coming from), so that we can review the reliability of our visual testing over time.